dymos/trajectory/trajectory.py

```diff
@@ -1018,27 +1013,30 @@ def _configure_solvers(self):
         These solvers can be changed through the
         'default_nonlinear_solver' and 'default_linear_solver' options.
         """
-        if self._has_connected_phases and MPI and self.comm.size > 1:
+        if not self.options['auto_solvers']:
+            return
 
+        warn = False
+        if self._has_connected_phases and isinstance(self.phases, om.ParallelGroup) and MPI and self.comm.size > 1:
```

Comment:
Since self.comm.size will never be > 1 if MPI is not active, you don't really need the `and MPI` in the if check.